components/atoms/ChooseImage.tsx

```diff
@@ -0,0 +1,92 @@
+import { Plus } from '@tamagui/lucide-icons'
+import * as ImagePicker from 'expo-image-picker'
+import React, { useState } from 'react'
+import { Alert, useColorScheme } from 'react-native'
+import { Button, Image, XStack } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+
+const ChooseImage = (): React.ReactElement => {
+  const [selectedImages, setSelectedImages] = useState<string[]>([])
+  const colors = getColors(useColorScheme())
+
+  const pickImages = async (): Promise<void> => {
+    // Yêu cầu quyền truy cập ảnh
+    const permissionResult =
+      await ImagePicker.requestMediaLibraryPermissionsAsync()
+
+    if (!permissionResult.granted) {
+      Alert.alert('Permission to access camera roll is required!')
+      return
+    }
+
+    // Mở trình chọn ảnh với giới hạn 3 ảnh
+    const result = await ImagePicker.launchImageLibraryAsync({
+      allowsMultipleSelection: true,
+      mediaTypes: ImagePicker.MediaTypeOptions.Images, // Cho phép chọn nhiều ảnh
+      selectionLimit: 3 // Giới hạn tối đa 3 ảnh
+    })
+
+    if (result.assets != null) {
+      const uris = result.assets.map((asset) => asset.uri)
+      setSelectedImages(uris)
+      // Gọi hàm upload cho từng ảnh
+      // uris.forEach(uri => uploadImage(uri));
+    }
+  }
+
+  //   const uploadImage = async (uri: string) => {
```

Comment:
Creating separate sub -components (e.g., component for each image or part of its own upload processing) can help the code easier to manage when expanding

Reply:
I will update in the future